nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -246,14 +266,15 @@ BtNavigator::navigateToPose()
       return action_server_->is_cancel_requested();
     };
 
-  auto bt_xml_filename = action_server_->get_current_goal()->behavior_tree;
+  std::string bt_xml_filename = action_server_->get_current_goal()->behavior_tree.c_str();
 
   // Empty id in request is default for backward compatibility
   bt_xml_filename = bt_xml_filename.empty() ? default_bt_xml_filename_ : bt_xml_filename;
+  RCLCPP_DEBUG(get_logger(), "Check if we need to load BT: %s", bt_xml_filename.c_str());
```

Comment:
Remove debug log

Reply:
why?

Serious question. I tried to get up the rabbit hole of the RCLCPP_* Macros and saw that it is ignored if the severity is not met..?

http://docs.ros2.org/latest/api/rcutils/logging__macros_8h.html#a887650903d2c7048a929cbb41a7c9d9a

So is it even creating any lag when the severity is not met? If yes.. of course, they will be deleted straight away. If no, then I find those interesting for future folks trying to get a grasp about what is happening here and there. Maybe not from this debug message specifically but from debug output in general (if done right)

Reply:
Yes, it populates log files. Debug will still go to log files even if not printing on screen, so this would spam your logs later every request. If trying to make what's happening inline of code, a comment is probably a better choice. A log is more for users at run-time.

Reply:
ah, thanks.. that makes sense.